_docs/guides/create-notes-occurrences.md

```diff
@@ -0,0 +1,260 @@
+---
+title: How to create notes and occurrences
+overview: Describes how to create notes and occurrences once you have a running Grafeas server.
+
+order: 15
+
+layout: docs
+type: markdown
+---
+
+Notes and occurrences are the basic building blocks of storing metadata in
+Grafeas. Creating and managing them allows you to track information about your
```

Comment:
Creating and managing them...: i think that line is a little distracting. should we delete it?

Reply:
Sure.